controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/Change.java

```diff
@@ -57,17 +60,32 @@ public boolean isPresent() {
     /** Returns the application version carried by this. */
     public Optional<ApplicationVersion> application() { return application; }
 
+    public boolean isPinning() { return pinning; }
```

Comment:
Nit-pick: Shouldn't this be called `isPinned`?

Reply:
Not sure. I considered both. If you prefer isPinned, I'll let that sway my opinion.